system/lib/fetch/emscripten_fetch.cpp

```diff
@@ -236,6 +237,12 @@ EMSCRIPTEN_RESULT emscripten_fetch_close(emscripten_fetch_t *fetch)
 static void fetch_free(emscripten_fetch_t *fetch)
 {
 	fetch->id = 0;
+	if(fetch->responseHeaders)
+	{
+		for(size_t i = 0; fetch->responseHeaders[i]; ++i)
+			free((void*)fetch->responseHeaders[i]);
+		free((void*)fetch->responseHeaders);
+	}
```

Comment:
The contents of a `emscripten_fetch_free_unpacked_response_headers(char **unpackedHeaders)` would contain this code.

Reply:
Done.